docs/getting-started/clients/mobile/android/index.md

```diff
@@ -5,13 +5,12 @@
 Before you start, you should have the recommended [Tools and Libraries](../../../tools/index.md)
 installed. You will also need to install:
 
-1.  Visual Studio 2019 (required for a dependency not included yet in VS 2022; development can be
-    done in VS 2022)
-2.  [dotnet Core 3.1 (latest)](https://dotnet.microsoft.com/en-us/download/dotnet/3.1)
+1.  Visual Studio 2022
+2.  [.NET 7 (latest)](https://dotnet.microsoft.com/en-us/download/dotnet/7.0)
```

Comment:
ℹ️ Technically not the latest anymore since 8 (LTS) is live. Probably not necessary to update given the change to the native apps.

Reply:
Roger that, I meant to say latest feature/patch inside the 7 version but yeah I agree.